backend/src/main/java/cloudflight/integra/backend/model/validation/ProposalValidation.java

```diff
@@ -0,0 +1,6 @@
+package cloudflight.integra.backend.model.validation;
+import cloudflight.integra.backend.model.Proposal;
+
+public interface ProposalValidation {
+    void validate(Proposal proposal) throws ProposalValidationError;
+}
```

Comment:
Great idea using an Interface here.
It would be a good idea if we were to turn this class into a generic one.
It seems like a good Interface to follow when implementing other validators.

```java
public interface GenericValidator<T> {
    void validate(T object) throws ValidationError;
}
```

We would have to create a generic Exception (`ValidationError`), which would be extended later by the `ProposalValidationError`